Frameworks/UIKit/UIView.mm

```diff
@@ -274,30 +260,106 @@ - (bool)_processGesturesForTouch:(UITouch*)touch event:(UIEvent*)event touchEven
         }
     }
 
-    // gesture priority list
-    const static id s_gesturesPriority[] = {[UIPinchGestureRecognizer class], [UISwipeGestureRecognizer class],
-                                            [UIPanGestureRecognizer class],   [UILongPressGestureRecognizer class],
-                                            [UITapGestureRecognizer class],   [_UIDMPanGestureRecognizer class] };
+    //
+    // After state transtion, we potentially get active gestures, we should remove any gestures that can not be recognized simultaneously
+    //
+    NSMutableSet* gesturesToRemove = [NSMutableSet set];
+    for (UIGestureRecognizer* curGesture in [g_currentlyTrackingGesturesList reverseObjectEnumerator]) {
+        UIGestureRecognizerState state = curGesture.state;
+        if (state != UIGestureRecognizerStatePossible && state != UIGestureRecognizerStateCancelled) {
```

Comment:
also UIGestureRecognizerStateEnded?

---
In reply to: [99709909](https://github.com/Microsoft/WinObjC/pull/1909#discussion_r99709909) [](ancestors = 99709909)